src/main/resources/default/taglib/t/lookupValue.html.pasta

```diff
@@ -21,21 +21,27 @@
                 suggestionUri="@apply('/system/lookuptable/autocomplete/%s/%s/%s', table, value.getDisplay(), value.getExtendedDisplay())">
     <i:block name="addon">
         <i:if test="showSelectionButton">
-            <a class="btn btn-outline-secondary" id="@id-addon"><i class="fa fa-bolt"></i></a>
+            <a class="btn btn-outline-secondary" id="@id-addon" tabindex="0"><i class="fa fa-bolt"></i></a>
         </i:if>
     </i:block>
     <i:block name="script">
         <i:if test="showSelectionButton">
             const helper = document.getElementById('___id-addon');
-            helper.addEventListener('click', function () {
-                new LookupTableInfo({
-                    tableName: '___table',
-                    labelFormat: '___value.getDisplay()',
-                    codeCallback: function (code, name) {
-                        autocomplete.val({value: code, text: name});
-                    }
-                });
+            helper.addEventListener('click', getLookupTableInfo);
+            helper.addEventListener('keydown', function (e) {
+                if (e.keyCode === 13) {
```

Comment:
Maybe but the 13 in a variable?

Reply:
Mhhh if we do so I would put a KeyCodes list in sirius, like `sirius.KeyCode.ENTER`

Reply:
👍 will follow in an extra PR after https://github.com/scireum/sirius-web/pull/1317 is merged.

Reply:
https://github.com/scireum/sirius-web/pull/1317 is merged now, the respective version of web is available in `develop`. Please merge `develop` into this PR's feature branch and modify this line accordingly.